src/main/java/io/cos/cas/osf/web/flow/configurer/OsfCasLoginWebflowConfigurer.java

```diff
@@ -320,6 +322,11 @@ private void createOsfDefaultLoginCheckAction(final Flow flow) {
                 OsfCasWebflowConstants.TRANSITION_ID_INSTITUTION_LOGIN,
                 OsfCasWebflowConstants.STATE_ID_OSF_INSTITUTION_LOGIN_CHECK
         );
+        createTransitionForState(
+                action,
+                OsfCasWebflowConstants.TRANSITION_ID_UNSUPPORTED_INSTITUTION_LOGIN,
+                OsfCasWebflowConstants.STATE_ID_OSF_UNSUPPORTED_INSTITUTION_LOGIN_CHECK
```

Comment:
As mentioned, this can go to `OsfCasWebflowConstants.VIEW_ID_UNSUPPORTED_INSTITUTION_SSO_INIT` since the `OsfUnsupportedInstitutionLoginPreparationAction` doesn't do anything extra.